PR build failed with
```
15:58:44 /sd/workspace/src/github.com/vespa-engine/vespa/vespalib/src/tests/nexus/nexus_test.cpp: In lambda function:
15:58:44 /sd/workspace/src/github.com/vespa-engine/vespa/vespalib/src/tests/nexus/nexus_test.cpp:70:9: error: ‘EXPECT_EQ’ was not declared in this scope
15:58:44    70 |         EXPECT_EQ(ctx.num_threads(2));
15:58:44       |         ^~~~~~~~~
```

--- a/vespalib/src/tests/nexus/nexus_test.cpp
+++ b/vespalib/src/tests/nexus/nexus_test.cpp
@@ -1,4 +1,4 @@
-// Copyright Yahoo. Licensd under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include <vespa/vespalib/test/nexus.h>
 #include <vespa/vespalib/gtest/gtest.h>

--- a/vespalib/src/tests/nexus/nexus_test.cpp
+++ b/vespalib/src/tests/nexus/nexus_test.cpp
@@ -67,7 +67,7 @@ TEST(NexusTest, example_multi_threaded_unit_test) {
     int a = 0;
     int b = 0;
     auto work = [&](Nexus &ctx) {
-        EXPECT_EQ(ctx.num_threads(2));
+        EXPECT_EQ(ctx.num_threads(), 2);
         if (ctx.thread_id() == 0) {
             a = 5;
             ctx.barrier();

--- a/vespalib/src/tests/nexus/nexus_test.cpp
+++ b/vespalib/src/tests/nexus/nexus_test.cpp
@@ -7,7 +7,7 @@
 using namespace vespalib::test;
 
 TEST(NexusTest, run_void_tasks) {
-    std::atomic<size_t> value;
+    std::atomic<size_t> value = 0;
     auto task = [&value](Nexus &) {
                     value.fetch_add(1, std::memory_order_relaxed);
                 };
@@ -19,7 +19,7 @@ TEST(NexusTest, run_void_tasks) {
 }
 
 TEST(NexusTest, run_value_tasks_select_thread_0) {
-    std::atomic<size_t> value;
+    std::atomic<size_t> value = 0;
     auto task = [&value](Nexus &ctx) {
                     value.fetch_add(1, std::memory_order_relaxed);
                     return ctx.thread_id() + 5;
@@ -30,7 +30,7 @@ TEST(NexusTest, run_value_tasks_select_thread_0) {
 }
 
 TEST(NexusTest, run_value_tasks_merge_results) {
-    std::atomic<size_t> value;
+    std::atomic<size_t> value = 0;
     auto task = [&value](Nexus &) {
                     return value.fetch_add(1, std::memory_order_relaxed) + 1;
                 };
@@ -40,6 +40,12 @@ TEST(NexusTest, run_value_tasks_merge_results) {
 }
 
 TEST(NexusTest, run_inline_voted_loop) {
+    // Each thread wants to run a loop <thread_id> times, but the loop
+    // condition is a vote between all threads. After 3 iterations,
+    // threads 0,1,2,3 vote to exit while threads 4,5,6,7,8 vote to
+    // continue. After 4 iterations, threads 0,1,2,3,4 vote to exit
+    // while threads 5,6,7,8 vote to continue. The result is that all
+    // threads end up doing the loop exactly 4 times.
     auto res = Nexus(9).run([](Nexus &ctx) {
                                 size_t times = 0;
                                 for (size_t i = 0; ctx.vote(i < ctx.thread_id()); ++i) {

